packages/cashier-v2/src/lib/WithdrawalVerification/WithdrawalVerificationRequest/WithdrawalVerificationRequest.scss

```diff
@@ -0,0 +1,13 @@
+.wallets-withdrawal-verification-request {
```

Comment:
so ultimately we are still on BEM or actually on modules?

Reply:
Oh. I forgot to migrate these files to scss modules 😓 
Will do that 🙏